website2/website/scripts/build-site.sh

```diff
@@ -30,5 +30,5 @@ yarn build
 node ./scripts/replace.js
 
 rm -rf ${ROOT_DIR}/generated-site/content
-mkdir -p ${ROOT_DIR}/generated-site/content
+mkdir -p ${ROOT_DIR}/generatedori-site/content
```

Comment:
this is expected?

Reply:
My Bad, Mistake. I was testing around.. Fixed it..